test/jdk/java/util/Calendar/Bug4766302.java

```diff
@@ -24,16 +24,19 @@
 /*
  * @test
  * @bug 4766302
- * @summary Make sure that computeTime call doesn't reset the isTimeSet value.
+ * @summary Make sure that calling computeTime doesn't reset the isTimeSet value.
+ * @run junit Bug4766302
  */
 
 import java.util.GregorianCalendar;
 
-@SuppressWarnings("serial")
```

Comment:
Is removing this OK?

Reply:
At first I thought so, there is no warning about a missing serialVersionUID when the _SuppressWarnings_ annotation is removed, and IntelliJ actually flags the annotation as redundant. But since it was added separately and intentionally in [JDK-8165296](https://bugs.openjdk.org/browse/JDK-8165296), I would rather leave it alone on second thought.

Reply:
Not sure why IntelliJ claims it redundant, but compiling the test case alone, without that `@SuppressWarnings` with `-Xlint` gives me this:
```
warning: [serial] serializable class MyCalendar has no definition of serialVersionUID
```